target/linux/ramips/image/mt7620.mk

```diff
@@ -468,6 +468,16 @@ define Device/hnet_c108
 endef
 TARGET_DEVICES += hnet_c108
 
+define Device/hootoo_tm05
+  MTK_SOC := mt7620n
+  IMAGE_SIZE := 7872k
+  DEVICE_VENDOR := HooToo
+  DEVICE_MODEL := TM05
+  DEVICE_PACKAGES := kmod-usb2 kmod-usb-ohci kmod-i2c-core kmod-i2c-ralink
+  SUPPORTED_DEVICES += tm05
```

Comment:
That's not required.

Reply:
Sorry, what's not required? The SUPPORTED_DEVICES line?

Reply:
Yes, it's calculated automatically from the node name.

Reply:
OK, removed.